docs/source/zenodo.rst

```diff
@@ -0,0 +1,97 @@
+Firedrake Zenodo integration: tools for reproducible science
+============================================================
+
+.. image:: _static/zenodo-gradient-1000.png
+   :width: 25%
+   :alt: Zenodo
+   :align: right
+   :target: https://zenodo.org
+   :class: round-corners
+            
+.. image:: _static/GitHub_Logo.png
+   :width: 25%
+   :alt: Zenodo
+   :align: right
+   :target: http://github.com
+   :class: round-corners
+
+
+`Zenodo <http://zenodo.org>`_ provides a facility for archiving
+scientific data, such as software. Zenodo provides secure archiving
+and referability, including digital object identifiers
+(DOIs). Firedrake integrates with Zenodo and GitHub to provide Firedrake users
+with the ability to generate a set of DOIs corresponding to the exact
+set of Firedrake components which were used to conduct a particular
+simulation.
+
+These DOIs can be used in citations in publications to provide a
+reference to the exact version of the software used, and thereby to
+improve the reproducibility of your computational science.
+
+How to register DOIs for a version of Firedrake
+-----------------------------------------------
+
+This section assumes that you have a Firedrake installation which you
+have used to conduct some numerical experiment and which you wish to
+publish or otherwise record for posterity. It is assumed that your
```

Comment:
I found this sentence a little hard to parse.  Unfortunately, in attempting to come up with something better I thus far have nothing.